### ISSUE TITLE
[BUG]: nested clozes problem

### ISSUE THREAD

**Issue by euu2021** [ankidroid/Anki-Android#14444]
### Checked for duplicates?

- [X] This issue is not a duplicate

### What are the steps to reproduce this bug?

In a note like this:

> a
{{c2::b {{c1::c}}d
{{c3::e}} }}

### Expected behaviour

All clozes working fine.

### Actual behaviour

Ankidroid says that c1 is Empty. The other cards show up normally.

On the desktop, c1 works fine.

### Debug info

```text
AnkiDroid Version = 2.16.5 (953b9bd879269910ee962b520da9705336d8bc2d)

Android Version = 13

ProductFlavor = play

Manufacturer = Xiaomi

Model = 21061110AG

Hardware = mt6893

Webview User Agent = Mozilla/5.0 (Linux; Android 13; 21061110AG Build/TP1A.220624.014; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/117.0.0.0 Mobile Safari/537.36

ACRA UUID = 7868d464-e23f-4df8-bc3c-262f71320c26

New schema = true

Scheduler = std3

Crash Reports Enabled = true

DatabaseV2 Enabled = true
```

### (Optional) Anything else you want to share?

Maybe, the problem is that there are two clozes on the same line?

### Research

- [X] I am reporting a bug specific to AnkiDroid (Android app)
- [X] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [ ] (Optional) I have confirmed the issue is not resolved in the latest alpha release ([instructions](https://docs.ankidroid.org/manual.html#betaTesting))

**Comment by dae** [comment]
Please let us know if 2.17 alpha doesn't fix this, when it comes out.

**Comment by lukstbit** [comment]
From my testing this seems to be fixed with the new backend which will be available starting with version 2.17alpha2.
I'm closing this for now as it's most likely fixed, if you see this behavior with the alpha version I mentioned please leave a comment and I'll reopen the issue.

**Comment by lukstbit** [comment]
@euu2021 It's not going to be one year but I doubt it's going to be less than a month. The new backend will be available starting with 2.17alpha2(which should hopefully be available soon).
